manual_jb/content/_toc.yml

```diff
@@ -17,6 +17,7 @@ chapters:
       - file: analyses/bridge_dmg
       - file: analyses/building_cluster_recovery
       - file: analyses/building_dmg
+      - file: analyses/building_structural_dmg
```

Comment:
This should be after building non structural damage so it's in alphabetical order